.gitignore

```diff
@@ -6,6 +6,7 @@
 /.build
 /.release
 /.tarballs
+/.idea
```

Comment:
please remove user-specific gitignore changes

Reply:
Why?  Isn't to good to ignore common things?

Reply:
By what definition of "common" ? :) we use our gitignore for project specific ignores; if you have your own files you need to ignore because of your environment, please use your global gitignore.

Reply:
Fair enough.  I'll remove this.